client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -51,6 +52,27 @@ module.exports = React.createClass( {
 		} );
 	},
 
+	monthlyPrice: function() {
+		const { cost, product_slug: productSlug, currency } = this.props.cartItem,
+			// A hacky way to check if we are in signup, since for some reason cartItem.extra.context
+			// is missing in that case (it seems cartItem.extra is overwritten somewhere between)
+			isInSignup = this.props.cartItem.extra && this.props.cartItem.extra.context !== 'calypstore',
```

Comment:
Thank you, @drewblaisdell, for the suggestion! But while pursuing it, I've noticed that _now_ we are receiving the proper `extra: { context: 'signup' }` part of the cart item (yup, from the server, too!) >_< I don't know if something changed in the backend (I've skimmed the logs and didn't see anything obviously related) or was it an oversight/error on my part or my tools.
So sorry for the false alarm, but it seems that it's working as it should. And I can just check for the proper `context`.
Could you (or @lucasartoni) check if the flow (most importantly the `YY EUR x 12 months` part in the cart item) is working properly for you too?